core/src/cluster_info_vote_listener.rs

```diff
@@ -37,22 +46,25 @@ impl ClusterInfoVoteListener {
         cluster_info: &Arc<RwLock<ClusterInfo>>,
         sigverify_disabled: bool,
         sender: &Sender<VerifiedPackets>,
+        poh_recorder: Arc<Mutex<PohRecorder>>,
     ) -> Result<()> {
         let mut last_ts = 0;
         loop {
             if exit.load(Ordering::Relaxed) {
                 return Ok(());
             }
             let (votes, new_ts) = cluster_info.read().unwrap().get_votes(last_ts);
-            last_ts = new_ts;
-            inc_new_counter_info!("cluster_info_vote_listener-recv_count", votes.len());
-            let msgs = packet::to_packets(&votes);
-            let r = if sigverify_disabled {
-                sigverify::ed25519_verify_disabled(&msgs)
-            } else {
-                sigverify::ed25519_verify_cpu(&msgs)
-            };
-            sender.send(msgs.into_iter().zip(r).collect())?;
+            if poh_recorder.lock().unwrap().bank().is_some() {
```

Comment:
@pgarg66 Why no test for this branch?  Also, why push all the original code into the branch and not do:

```rust
if poh_recorder.lock().unwrap().bank().is_none() {
    continue;
}
```